float8_experimental/config.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD 3-Clause license found in the
+# LICENSE file in the root directory of this source tree.
+
+# If True, allocates buffers for float8 weight cache
+allocate_float8_weight_cache_buffers = False
```

Comment:
I think some more comments on how users are expected to use this in code

Reply:
added to readme